utils/utils.go

```diff
@@ -419,26 +500,6 @@ func AtTip(
 	return blockTimestamp >= tipCutoff
 }
 
-// CheckAtTip returns a boolean indicating if a
-// Rosetta implementation is at tip.
-func CheckAtTip(
-	ctx context.Context,
-	networkIdentifier *types.NetworkIdentifier,
-	helper FetcherHelper,
-	tipDelay int64,
-) (bool, error) {
-	status, fetchErr := helper.NetworkStatusRetry(
-		ctx,
-		networkIdentifier,
-		nil,
-	)
-	if fetchErr != nil {
-		return false, fmt.Errorf("%w: unable to get network status", fetchErr.Err)
-	}
-
-	return AtTip(tipDelay, status.CurrentBlockTimestamp), nil
-}
```

Comment:
This method is not used by the SDK or the CLI and we are assuming no one else is using it either.